src/test/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+hmpps:
```

Comment:
Does this not need to be defined also in the main application.yml?
How would this placeholder ${hmpps.auth.url} be interporlated?
Just a question for my understanding, as this might work and be fine.

Reply:
No need then if it's directly translating hmpps.auth.url to the value of HMPPS_AUTH_URL in the env vars.
Thanks